File: Todorin/Todorin/Todorin/Models/Priority.cs
namespace Todorin.Models
{
    public class Priority
    {
        public Priority()
        {
            PrioritySort = 0;
        }

        public string AppUserId { get; set; }
        public string PriorityName { get; set; }
        public int PrioritySort { get; set; }
        public string SyncDt { get; set; }
        public string Id { get; set; }
    }
}

File: flut_todo/lib/models/priority.dart
class Priority {
  String? appUserId;
  String priorityName;
  int prioritySort;
  String? syncDt;
  String? id;

  Priority(
      {this.appUserId,
      required this.priorityName,
      this.prioritySort = 0,
      this.syncDt,
      this.id});

  factory Priority.fromJson(Map<String, dynamic> json) {
    return Priority(
        appUserId: json['appUserId'],
        priorityName: json['priorityName'],
        prioritySort: json['prioritySort'],
        syncDt: json['syncDt'],
        id: json['id']);
  }
}
